configs/biggan/README.md

```diff
@@ -0,0 +1,82 @@
+# LARGE SCALE GAN TRAINING FOR HIGH FIDELITY NATURAL IMAGE SYNTHESIS
+
+## Introduction
+<!-- [ALGORITHM] -->
+```latex
+@inproceedings{
+brock2018large,
+title={Large Scale {GAN} Training for High Fidelity Natural Image Synthesis},
+author={Andrew Brock and Jeff Donahue and Karen Simonyan},
+booktitle={International Conference on Learning Representations},
+year={2019},
+url={https://openreview.net/forum?id=B1xsqj09Fm},
+}
+```
+
+The BigGAN/BigGAN-Deep is a conditional generation model that can generate both high-resolution and high-quality images by scaling up the batch size and number of model parameters.
+
+We have conducted training BigGAN with CIFAR10(3x32x32) and ImageNet1k(3x128x128) dataset, and the sampling results are showed below.
```

Comment:
We have finished training BigGAN in Cifar10(32x32) and are aligning training performance in ImageNet1k (128x128). Some sampled results are shown below for your reference.